mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -202,6 +223,7 @@ def run(self):  # {{{
             #                                      sectionName, dictClimo,
             #                                      dictTseries)
         else:
+            self._load_mesh()
```

Comment:
It makes more sense to load the mesh variables only once and to store the results for later use.  I guess the only reason not to do that would be if there are memory issues, but these aren't big variables compared to others we load.